### DISCUSSION TITLE
Add repeat/loop function to Jukebox

### DISCUSSION THREAD

**Discussion by g0urra** [P3pp3rF1y/SophisticatedBackpacks#317] [Ideas]
Like the title says, it would be great to see a loop/repeat functionality to the Jukebox Upgrade so I won't have to go into the backpack every time to repeat a song. Maybe also an optional pause between the loops.

**Comment by w1kee** [comment]
This would also fix the issue of Quark's Ambience discs not looping like they're intended to.

**Comment by SquareBall789** [comment]
Also an upgraded version that has a 3x3 inventory size for discs which will play at random when the last disc track is over?

**Reply by xmightymeatballx** [reply]
Only if the shuffle button is enabled :D

**Comment by NoBar94** [comment]
To go off of these comments, adding more buttons in general to the Jukebox tab! For example a repeat button, a shuffle button that selects discs at random from within the backpack, a rewind button, etc. Honestly if it were possible I would go so far as to dedicate hotkeys to these functions.

**Comment by konek101** [comment]
yess! add it!

**Comment by ProGmr1337** [comment]
Also add mono audio for jukebox upgrade, cause when i look up and down it sounds different. Ex. headphones, pair them to a backpack and give mono audio (and add a curios slot for them, or not)
